package.json

```diff
@@ -108,9 +108,9 @@
     "react-dropzone": "14.2.9",
     "react-is": "18.3.1",
     "react-window": "1.8.10",
-    "remark": "14.0.3",
-    "remark-gfm": "1.0.0",
-    "remark-react": "9.0.1",
+    "remark": "15.0.1",
+    "remark-gfm": "4.0.0",
```

Comment:
ESM modules (introduced in `remark-gfm` v2) are [not yet supported by Gatsby in TS](https://www.gatsbyjs.com/docs/how-to/custom-configuration/es-modules/#current-limitations)